web/src/javascript/app.js

```diff
@@ -1,13 +1,15 @@
 import fakeState from "./fakeState";
-import {draw as drawGame} from "./game";
+import {draw as drawGame, fetchState} from "./game";
 import {drawLeaderboard} from "./gameLeaderboard";
 import {createGameCanvas} from "./drawMap";
 
 const gameCanvas = createGameCanvas();
+let currentState = fakeState.data;
 
-function updateLoop() {
-  drawGame(fakeState.data, gameCanvas);
-  drawLeaderboard(fakeState.data);
+async function updateLoop() {
+  currentState = await fetchState(0);
```

Comment:
Hmm it's fine, let's handle it once we do private games. I think we need to `fetchState(_the right ID over here_)`, because we "don't care" about the ranked game when looking at the private game, for example. Anyway, all good for this PR. Let's try to find a nice solution along the way. :)